src/main/resources/application.yaml

```diff
@@ -1,6 +1,19 @@
 server:
   port: 5022
 
+spring:
+  jpa:
+    database: POSTGRESQL
+    show-sql: false
+    hibernate:
+      ddl-auto: validate
+      naming:
+        physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
+        implicit-strategy: org.hibernate.boot.model.naming.ImplicitNamingStrategyJpaCompliantImpl
+    properties:
+      dialect: org.hibernate.dialect.PostgreSQLDialect
+    database-name: actions
```

Comment:
to move like in the latest study-server pr

Reply:
Done